23 Realizzazione dell'entità SkillEmp

mate94 opened the pull request:
Essa comprende la realizzazione di tutte le classi utili alla gestione della suddetta entità:
controller, service, repository ed entità

Closes #23

mate94 committed:
Realizzazione dell'entità SkillEmp effettuata
Essa comprende la realizzazione di tutte le classi utili alla gestione della suddetta entità:
controller, service, repository ed entità

Wabri commented on src/main/java/plansoft/skillsmanagement/skillemp/SkillEmpService.java:
```diff
@@ -0,0 +1,87 @@
+
+
+	//change level of a skill related to an employee
+	public void updateSkillEmpLevel(int id_lev,int id_emp, int id_skill) {
```
Camel case

Wabri reviewed (changes requested):
Riguardare i nomi in generale, cercare di essere il più chiaro possibile, pensare sempre a chi deve leggere il nostro codice.

emp = https://en.wikipedia.org/wiki/Electromagnetic_pulse

mate94 committed:
Codice editato

Modifiche apportate su nomi di variabili e metodi con lo scopo di
rendere il codice più chiaro possibile. Utilizzo del camel-case dove
possibile

LORYMAX4 commented on src/main/java/plansoft/skillsmanagement/skillemployee/SkillEmployee.java:
```diff
@@ -0,0 +1,60 @@
+	@Id
+	private int id;
+
+	private int idemployee;
```
Camel case

LORYMAX4 commented on src/main/java/plansoft/skillsmanagement/skillemployee/SkillEmployee.java:
```diff
@@ -0,0 +1,60 @@
+	private int id;
+
+	private int idemployee;
+	private int idskill;
```
Camel case

LORYMAX4 commented on src/main/java/plansoft/skillsmanagement/skillemployee/SkillEmployee.java:
```diff
@@ -0,0 +1,60 @@
+
+	private int idemployee;
+	private int idskill;
+	private int idlevel;
```
Camel case

LORYMAX4 commented on src/main/java/plansoft/skillsmanagement/skillemployee/SkillEmployeeRepository.java:
```diff
@@ -0,0 +1,15 @@
+
+public interface SkillEmployeeRepository extends CrudRepository<SkillEmployee, Integer> {
+
+	List<SkillEmployee> findAllByIdskill(int idskill);
```
Camel case

LORYMAX4 commented on src/main/java/plansoft/skillsmanagement/skillemployee/SkillEmployeeRepository.java:
```diff
@@ -0,0 +1,15 @@
+
+	List<SkillEmployee> findAllByIdskill(int idskill);
+
+	List<SkillEmployee> findAllByIdemployee(int id_emp);
```
Vedi sopra

LORYMAX4 commented on src/main/java/plansoft/skillsmanagement/skillemployee/SkillEmployeeRepository.java:
```diff
@@ -0,0 +1,15 @@
+
+	List<SkillEmployee> findAllByIdemployee(int id_emp);
+
+	void deleteByIdemployee(int id_emp);
```
Vedi sopra

LORYMAX4 reviewed (changes requested):
Camel case da rivedere

mate94 committed:
Camel Case

utilizzo camel case nella repository dell'entità SkillEmployee e nella
classe SkillEmployee
